Composer/plugins/azurePublish/src/deploy.ts

```diff
@@ -55,8 +55,9 @@ export class BotProjectDeploy {
 
       // STEP 2: UPDATE LUIS
       // Do the LUIS build if LUIS settings are present
-      if (!language) {
-        language = 'en-us';
+      let luisLanguage = language || settings.luis.defaultLanguage;
```

Comment:
I think we could just take defaultLanguage. you can update the code like this:

let language = defaultLanguage || settings.luis.defaultLanguage；
if (!language){
   language = 'en-us'
}